requirements.txt

```diff
@@ -1 +1,4 @@
 zpl2>=1.1
+pycups==1.9.73
+PyPDF2==1.18
+requests
```

Comment:
IMHO, it should be great if you specify a version here.
Odoo core is having its version requests https://github.com/odoo/odoo/blob/12.0/requirements.txt#L40 and your module could override it in an unwanted manner.

Regards

Reply:
@flotho so, if it's already in Odoo's requirements, IMHO it should be removed from here. It makes no sense to re-define it from here. Don't you think?

The same applies for PyPDF2, where the version specified in Odoo is even greater.

Reply:
Do we really need to pin requirements here? It's preferable to test with the latest version of the the dependencies, and specify a minimum required version in `requirements.txt`.